src/chat/FetchError.js

```diff
@@ -0,0 +1,35 @@
+/* @flow strict-local */
+
+import React, { PureComponent } from 'react';
```

Comment:
This file was copied from `NoMessages`; if anything looks weird, that's a likely reason.